### PR TITLE
View undo and redo list in GUI

### PR DESCRIPTION
**Pull request by tantantan277**

### CODE REVIEW

File: src/main/java/seedu/address/model/ModelManager.java [hunk 1]
```diff
@@ -88,6 +89,7 @@ public void setAddressBookFilePath(Path addressBookFilePath) {
     @Override
     public void setAddressBook(ReadOnlyAddressBook addressBook) {
         versionedAddressBook.resetData(addressBook);
+        undoableCommand = "clear";
```
Comment (shanseet) [inline src/main/java/seedu/address/model/ModelManager.java:99]: Maybe should this clear the undoableCommand list instead, when the data is reset? If not, will the list ever be cleared?

Comment (tantantan277) [reply src/main/java/seedu/address/model/ModelManager.java:99]: When the data is reset the "undo" command still can access the previous state so it shouldn't be a problem. The list is only cleared when the program is turned off or we undo everything in the list

Comment (ssunil3232) [reply src/main/java/seedu/address/model/ModelManager.java:99]: Think it's good to go

**Review by shanseet — COMMENTED** [review]
Looks good. Maybe could add in something in place of the browser!

**Review by weizhang05 — APPROVED** [review]
PR approved.

Note: Un-commented codes can be removed at a later stage.

**Pull request merged**